src/electron/externalMedia.html

```diff
@@ -0,0 +1,85 @@
+<html>
```

Comment:
Is this being run through Prettier? It reformatted on my system when ran prettier.

To avoid the same shenanigans we had with `package.json` it would probably be good to ensure `.html` files are being run through the same formatter in the same way across all our systems.